### PR TITLE
device-samsung-manta: Enable touchscreen input

### PR DESCRIPTION
**Pull request by BakaOsaka**
Adds touchscreen support to device-samsung-manta (Google Nexus 10) with assistance from ```/dev/ponies``` on Riot.
Tested successfully on the Weston demo .
---
[x] Merge on GitHub (see <https://postmarketos.org/merge>)

### CODE REVIEW

File: aports/device/device-samsung-manta/90-android-touch-dev.rules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Touchscreen
+SUBSYSTEM=="input", ATTRS{name}=="Atmel maXTouch Touchscreen", \
+ENV{ID_INPUT}="1", ENV{ID_INPUT_TOUCHSCREEN}="1"
```
Comment (MayeulC) [inline aports/device/device-samsung-manta/90-android-touch-dev.rules]: Is this file really necessary? It should be automatically generated with the device package (see https://github.com/postmarketOS/pmbootstrap/pull/1554#discussion_r195552363 right, @ollieparanoid ?)

Otherwise, that's nitpicking, but you [miss a newline](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline) at the end of file

File: aports/device/device-samsung-manta/APKBUILD [hunk 2]
```diff
@@ -12,11 +12,11 @@ makedepends="devicepkg-dev"
 source="deviceinfo"
 
 build() {
-	devicepkg_build $startdir $pkgname
+       devicepkg_build $startdir $pkgname
```
Comment (PureTryOut) [inline aports/device/device-samsung-manta/APKBUILD]: We use tabs for indentation here, please revert this change (same for the one below)

File: aports/device/device-samsung-manta/APKBUILD [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Reference: <https://postmarketos.org/devicepkg>
 pkgname="device-samsung-manta"
 pkgdesc="Google Nexus 10"
-pkgver=0.1
+pkgver=0.3
```
[the remark this answers is not in the document]
Comment (BakaOsaka) [reply aports/device/device-samsung-manta/APKBUILD]: My mistake - I incremented this twice, once for each commit. I realize since this will get squashed there's no need to do so. I've set it back to 0.2.

File: aports/device/device-samsung-manta/APKBUILD (hunk 2 above)
Comment (BakaOsaka) [reply aports/device/device-samsung-manta/APKBUILD]: Fixed in latest commit.
